restrict access to API #24

jbmeerkat opened the pull request.

jbmeerkat committed:
restrict access to API #24

zzet commented:
@jbmeerkat а во фронте ничего менять не нужно? Или мы пока не ходим на эти апишки?

zzet commented on app/controllers/api/v1/applications_controller.rb:
```diff
@@ -21,6 +23,14 @@ def show
+      end
+
+      def current_user
+        @current_user ||= User.find_by token: api_token
```
Наверное нельзя использовать этот токен:
https://github.com/lenshq/lens_server/blob/master/app/commands/register_github_user.rb#L24

jbmeerkat replied:
@zzet да, не очень хорошо =/ сделаю отдельный API token

jbmeerkat replied:
@zzet готово

jbmeerkat commented:
> а во фронте ничего менять не нужно? Или мы пока не ходим на эти апишки?
> @zzet неа

jbmeerkat committed:
use api_token instead of github token

add migration to update existing users
actualize test
remove useless field from factory

jbmeerkat committed:
remove deprecated helper

change deprecated 'before_filter' to 'before_action'

jbmeerkat committed:
move token generation to rake task

JanaVPetrova commented:
:+1:

zzet commented on spec/factories/user.rb:
```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :user do
```
почему пустая?

jbmeerkat replied:
@zzet потому, что он нее ничего пока не требуется кроме генерации пустых пользователей
